### ISSUE TITLE
Number of items in thumbnails page.

### ISSUE THREAD

**Issue by saudiqbal** [beamtic/php-photo-gallery#3]
Not an issue just a suggestion. It would be nice to specify how many thumbnails per page should be displayed. If I have 200 pictures with thumbnails displaying on one page it will be slower. Like nine thumbnails per page would be nice.

**Comment by jacobkri** [comment]
I agree, pagination would be nice. I will see if I can implement this in the next update :-) Thanks for the feedback.

**Comment by saudiqbal** [comment]
Finally managed to install it [on my home page](http://www.saudiqbal.com)

**Comment by saudiqbal** [comment]
Tried to add pagination https://github.com/saudiqbal/php-photo-gallery

**Comment by saudiqbal** [comment]
Forked a new version and added pagination and mobile responsiveness.

https://github.com/saudiqbal/PHP-photo-gallery-with-pagination

**Comment by Turbulentarius** [comment]
This is now done. Sorry for the delay. I've been so busy with real life stuff
